### PR TITLE
Fixed speed according to frsky protocol

### PR DESCRIPTION
**Pull request by SamuelBrucksch**
changed speed from cm/s to knots

### CODE REVIEW

**Comment by SamuelBrucksch** [comment]
The casting to uint is wrong, need to be a bracket before:
serialize16((uint16_t(GPS_speed * 0.0194384449)*100)%100);
has to be
serialize16(uint16_t((GPS_speed * 0.0194384449)*100)%100);

**Comment by hydra** [comment]
Please fix the compilation errors via another commit.

Thanks for the pull request.

**Comment by SamuelBrucksch** [comment]
It compiles now. Sorry for trouble, just did a quick try from work :)

File: src/main/telemetry/frsky.c [hunk 1]
```diff
@@ -242,11 +242,13 @@ static void sendSpeed(void)
     if (!STATE(GPS_FIX)) {
         return;
     }
-    //Speed should be sent in m/s (GPS speed is in cm/s)
+    //Speed should be sent in knots (GPS speed is in cm/s)
     sendDataHead(ID_GPS_SPEED_BP);
-    serialize16((GPS_speed * 0.01 + 0.5));
+    //convert to knots: 1cm/s = 0.0194384449 knots
+    serialize16((uint16_t)(GPS_speed * 0.0194384449));
```
Comment (ledvinap) [inline src/main/telemetry/frsky.c]: Using 
```
//convert to knots: 1cm/s = 0.0194384449 knots
serialize16(GPS_speed * 1944 / 10000);
serialize16((GPS_speed * 1944 / 100) % 100);
```
should be a bit faster ...

**Pull request merged**